### ISSUE TITLE
Training with multiple parameters

### ISSUE THREAD

**Issue by kendoll30** [idealo/image-super-resolution#34]
Hello,
Can we run the Training of the model with a range of parameters so it can generate multiple models and just select the best result and discard the rest.
Idditionally I am thinking of crowdsourcing for the best performance parameters ( with a top).

Does this make any sense?

Br
Vlad

**Comment by cfrancesco** [comment]
Yes, it does make a lot of sense, it is kind of the purpose of this project. 
There is no automatic way to do cross experiment selection, but set up a bunch of config files and you should be good to go.

Feel free add a PR with you best configuration and/or model weights!
